vg/draw/canvas.go

```diff
@@ -32,6 +32,9 @@ type TextStyle struct {
 
 	// Font is the font description.
 	Font vg.Font
+
+	// Rotation is the text rotation in radians.
```

Comment:
State which direction 0 is and which orientation the rotation is - both vary depending on context.